sequelize-example/README.md

```diff
@@ -0,0 +1,50 @@
+# sequelize-example
+
+[![build][build]][build-url]
+[![coverage][cover]][cover-url]
+
+The egg example project that uses egg-sequelize plugin.
+
+## QuickStart
+
+<!-- add docs here for user -->
+
+see [egg docs][egg] for more detail.
+
+### Unittest
+```shell
+$ createdb example-unittest --owner postgres
```

Comment:
add to `test/.setup.js` ? like https://github.com/eggjs/egg-mysql/blob/master/test/.setup.js